### PR TITLE
refactor administration part

### PR DESCRIPTION
**Pull request by whitewum**
as title

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/3. server-administration/storage-service-administration/job-manager.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ nebula> SUBMIT JOB COMPACT;
 | 40         |
 --------------
 
+> 请参考 storage-config 一节修改默认 compaction 线程数量。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/3. server-administration/storage-service-administration/job-manager.md:19]: add link to storage config section

**Comment by Amber1990Zhang** [comment]
Please change the links in the README.md TOC and the `yaml` file to make sure there are no broken links in the doc site.

**Comment by whitewum** [comment]
It's ready to review now

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 2]
```diff
@@ -198,7 +161,7 @@ Heartbeat failed, status:RPC failure in MetaClient: N6apache6thrift9transport19T
 
 # meta 日志
 Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
-E0415 22:32:38.944437 15532 AsyncServerSocket.cpp:762] failed to set SO_REUSEPORT on async server socket Protocol not available
+E0415 22:32:38.944437 15532 AsyncServerSocket.cpp:762] failed to set `SO_REUSEPORT` on async server socket Protocol not available
```
Comment (Amber1990Zhang) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md:164]: I don't think a backquote is necessary here. This is how the log actually looks

File: docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md [hunk 3]
```diff
@@ -29,7 +29,7 @@
 
      * [Ubuntu 1604](https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/${release_version}/nebula-${release_version}.ubuntu1804.amd64.deb)
 
-     * 链接中 `${release_version}` 为具体的发布版本号，例如要下载 1.0.0-rc2 Centos 7 的安装包，那么可以直接通过命令下载。
+     * 链接中 `${release_version}` 为具体的发布版本号，例如要下载 1.0.0-rc4 Centos 7 的安装包，那么可以直接通过命令下载。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md:32]: the link below is rc2

File: docs/manual-CN/3.build-develop-and-administration/4.storage-service-administration/job-manager.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
-# 作业管理（Job Manager）
+# 作业管理（flush 和 compact）
 
-作业特指在存储层运行的一些长任务。比如 `compact` 和 `flush`。管理指对作业进行管理。比如让作业排队执行、查看作业状态、停止作业、恢复作业等。
+作业特指在存储层运行的一些长任务。比如 `compact`，`flush`等。管理指对作业进行管理。比如让作业排队执行、查看作业状态、停止作业、恢复作业等。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/4.storage-service-administration/job-manager.md:3]: 、

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md (hunk 2 above)
Comment (whitewum) [reply docs/manual-CN/1.overview/2.quick-start/2.FAQ.md:164]: I prefer to highlight the keyword.
But it's ok to both

File: docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md (hunk 3 above)
Comment (whitewum) [reply docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md:32]: good catch

File: docs/manual-CN/3.build-develop-and-administration/4.storage-service-administration/job-manager.md (hunk 4 above)
Comment (whitewum) [reply docs/manual-CN/3.build-develop-and-administration/4.storage-service-administration/job-manager.md:3]: right

File: docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md [hunk 5]
```diff
@@ -29,10 +29,10 @@
 
      * [Ubuntu 1604](https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/${release_version}/nebula-${release_version}.ubuntu1804.amd64.deb)
 
-     * 链接中 `${release_version}` 为具体的发布版本号，例如要下载 1.0.0-rc2 Centos 7 的安装包，那么可以直接通过命令下载。
+     * 链接中 `${release_version}` 为具体的发布版本号，例如要下载 `1.0.0-rc4` `Centos 7.5` 的安装包，那么可以直接通过命令下载。
 
      ```bash
-     $ wget https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/1.0.0-rc2/nebula-1.0.0-rc2.el7-5.x86_64.rpm
+     $ wget https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/1.0.0-rc4/nebula-1.0.0-rc4.el7-5.x86_64.rpm
```
Comment (Shinji-IkariG) [inline docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md:35]: The oss download domain name in the Chinese document can be changed to oss-cdn.nebula-graph.com.cn
The oss download domain name in the English document can be changed to oss-cdn.nebula-graph.io

Comment (whitewum) [reply docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md:35]: @Amber1990Zhang can you help me to submit a new pr for this url

**Pull request merged**